app/src/main/java/com/launchdarkly/hello_android/MainActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.launchdarkly.hello_android
+
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import com.launchdarkly.hello_android.MainApplication.Companion.LAUNCHDARKLY_MOBILE_KEY
+import com.launchdarkly.sdk.android.LDClient
+
+class MainActivity : AppCompatActivity() {
+
+    // Set BOOLEAN_FLAG_KEY to the feature flag key you want to evaluate.
+    val BOOLEAN_FLAG_KEY = "my-boolean-flag"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        val textView : TextView = findViewById(R.id.textview)
+
+        if (BOOLEAN_FLAG_KEY == "my-boolean-flag") {
+            val builder = AlertDialog.Builder(this)
+            builder.setMessage("BOOLEAN_FLAG_KEY was not customized for your account.")
+            builder.create().show()
+        }
+
+        if (LAUNCHDARKLY_MOBILE_KEY == "mobile-key-from-launch-darkly-website") {
+            val builder = AlertDialog.Builder(this)
+            builder.setMessage("LAUNCHDARKLY_MOBILE_KEY was not customized for your account.")
```

Comment:
Shall we use `for this application` instead? So the customer is not confused about their LaunchDarkly account?

Reply:
Updated to use your proposed string.  Done